dockers/docker-database/base_image_files/sonic-db-cli

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+import os
+import sys
+import swsssdk
+import logging
+import subprocess
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.NullHandler())
+
+DOCKER_EXEC_FLAGS = "i"
+
+# Determine whether stdout is on a terminal
+if os.isatty(1):
+    DOCKER_EXEC_FLAGS += "t"
+
+if len(sys.argv) < 3:
+    msg = "More than three arguments are required. 'usage: sonic-db-cli 'dbname' 'redis commands''"
+    print >> sys.stderr, msg
+    logger.error(msg)
+else:
+    try:
+        dbid = swsssdk.SonicDBConfig.get_dbid(sys.argv[1])
+        dbport = swsssdk.SonicDBConfig.get_port(sys.argv[1])
+        cmd = "docker exec -{} database redis-cli -n {} -p {} {}".format(DOCKER_EXEC_FLAGS, dbid, dbport, "
+".join(sys.argv[2:]))
+        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True, stderr=subprocess.STDOUT)
+        print proc.stdout.read()
+    except RuntimeError:
```

Comment:
>except RuntimeError [](start = 4, length = 19)

make `try` block as minimal as possible, I guess you just need to include first 2 statements.

Reply:
modified